docs/deployment/composer-dependency.md

```diff
@@ -1,151 +1,38 @@
-  # With Drupal VM as a Composer dependency:
-  - "{{ config_dir }}/../examples/scripts/configure-solr.sh"
-```
-
-### Create a delegating `Vagrantfile`
-
-Create a delegating `Vagrantfile` that will catch all your `vagrant` commands and send them to Drupal VM's own `Vagrantfile`. Place this file in your project's root directory.
-
-```ruby
-# The absolute path to the root directory of the project. Both Drupal VM and
-# the config file need to be contained within this path.
-ENV['DRUPALVM_PROJECT_ROOT'] = "#{__dir__}"
-# The relative path from the project root to the config directory where you
-# placed your config.yml file.
-ENV['DRUPALVM_CONFIG_DIR'] = "config"
-# The relative path from the project root to the directory where Drupal VM is located.
-ENV['DRUPALVM_DIR'] = "vendor/geerlingguy/drupal-vm"
-
-# Load the real Vagrantfile
-load "#{__dir__}/#{ENV['DRUPALVM_DIR']}/Vagrantfile"
+```sh
+composer create-project drupal-composer/drupal-project:8.x-dev <some-dir> --stability dev --no-interaction
 ```
 
-When you issue `vagrant` commands anywhere in your project tree this file will be detected and used as a delegator for Drupal VM's own Vagrantfile.
+### Add Drupal VM as a Composer dependency
 
-Your project structure should now look like this:
+Require Drupal VM as a development dependency to your project.
 
-```
-├── Vagrantfile
-├── composer.json
-├── config/
-│   ├── config.yml
-│   ├── local.config.yml
-│   └── Vagrantfile.local
-├── docroot/
-│   ├── ...
-│   └── index.php
-└── vendor/
-    ├── ...
-    └── geerlingguy/
-        └── drupal-vm/
-```
+    composer require --dev geerlingguy/drupal-vm
 
-### Provision the VM
+_This command will scaffold a `Vagrantfile` in the root of your project. Feel free to add it to your `.gitignore` as the file is now managed by Drupal VM and will be reset each time you run `composer update`._
 
-Finally provision the VM using the delegating `Vagrantfile`.
+### Create a `config.yml` to configure the VM
 
-```sh
-vagrant up
-```
+Create and configure a config.yml file, eg. in `vm/config.yml`. You'll need at least the following variables:
 
-_Important: you should never issue `vagrant` commands through Drupal VM's own `Vagrantfile` from now on. If you do, it will create a secondary VM in that directory._
+    # Change the build strategy to use a composer.json file.
+    drupal_build_composer: true
+    drupal_build_composer_project: false
 
-## Drupal VM in a subdirectory without composer
+    # Tell Drupal VM that the composer.json file already exists and doesn't need to be transfered.
+    drupal_composer_path: false
 
-If you're not using `composer` in your project you can still download  Drupal VM (or add it as a git submodule) to any subdirectory in your project. As an example let's name that directory `box/`.
+    # Set the location of the composer.json file and Drupal core.
+    drupal_composer_install_dir: "/var/www/drupalvm"
+    drupal_core_path: "{{ drupal_composer_install_dir }}/web"
```

Comment:
Would it be too much to print this in the console if Drupal VM is added and no prior `Vagrantfile` exists? It would just make getting started so much easier if you wouldn't have to visit the docs...